src/11_args.py

```diff
@@ -73,4 +89,4 @@ def f2(*args):
 }
 
 # What thing do you have to add to make this work?
-f4(d)
+# f4(d)
```

Comment:
Similar thing here. f4 is taking in a variable amount of keys and values. but d is just an object. We need to signal to the function that we are passing in a variable which is an object instead of hard coded key/value pairs. 

`f4(**d)`